### PR TITLE
Add Malay translation

### PR DESCRIPTION
**Pull request by od3n**

### CODE REVIEW

**Comment by miyurusankalpa** [comment]
merged, thanks #1 👍
have a great hacktoberfest 😃

File: lang.ms.php [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+// Upload Carousel
+$lang['UPLOAD_CAR_LP'] = 'Laman Utama';
+$lang['UPLOAD_CAR_1'] = 'Tiada IP direkodkan semasa muat naik.';
+$lang['UPLOAD_CAR_2'] = 'Kami mengawal muat naik anda.';
+$lang['UPLOAD_CAR_3'] = 'Tiada iklan dibenarkan.';
+
+// Footer
+$lang['FOOTER_CONTACT'] = 'hubungi';
+$lang['FOOTER_API'] = 'api';
```
Comment (codegateau) [inline lang.ms.php]: Leave it as API not api . Because api when translated from Malay to English is 'fire'

File: lang.ms.php [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+$lang['FOOTER_BLOG'] = 'blog';
+$lang['FOOTER_TOS'] = 'terma';
+$lang['FOOTER_ABOUT_US'] = 'mengenai kami';
+
+// Stats
+$lang['STATS_HEAD'] = 'Statistik';
+
+// Image View Page
+$lang['IMG_VP_IP_HEAD'] = 'Prebiu Gambar';
```
Comment (codegateau) [inline lang.ms.php]: Preview in Malay is better translated as pratonton .

File: lang.ms.php [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+$lang['IMG_VP_SHARE_HEAD'] = 'Kongi gambar ini';
+$lang['IMG_VP_SHARE_DL'] = 'Muat Turun';
+$lang['IMG_VP_SHARE_DL_BTN_IMG'] = 'Gambar';
+$lang['IMG_VP_SHARE_DL_BTN_ZIP'] = 'Zip';
```
Comment (codegateau) [inline lang.ms.php]: Please leave it as ZIP .

File: lang.ms.php (hunk 1 above)
Comment (miyurusankalpa) [inline lang.ms.php]: no need to capitalize it, this is how it looks in the orginal

File: lang.ms.php (hunk 2 above)
Comment (miyurusankalpa) [inline lang.ms.php]: this change is welcome

File: lang.ms.php (hunk 3 above)
Comment (miyurusankalpa) [inline lang.ms.php]: no need to capitalize it, this is how it looks in the original

**Comment by miyurusankalpa** [comment]
@codegateau thanks for the review

**Pull request merged**